modules/apps/product-navigation/product-navigation-site-administration/src/main/resources/META-INF/resources/sites/site_administration_header.jsp

```diff
@@ -24,7 +24,7 @@ int childPanelCategoriesSize = GetterUtil.getInteger(request.getAttribute("produ
 
 				<aui:style type="text/css">
 					.site-administration--header {
-						background-image: url(<%= siteAdministrationPanelCategoryDisplayContext.getLogoURL() %>);
+						background-image: url(<%= siteAdministrationPanelCategoryDisplayContext.getLogoURL() %>) !important;
```

Comment:
Can we just inline this style on `site-administration--header-no-collapsible` so we don't need this class? We also wouldn't need `!important`.

Reply:
Hey @pat270 ... No, sadly not. We no longer allow for any inline styles in order to support the NONCE directive. This was caused by trying to follow this [LPD-50387](https://github.com/liferay/liferay-portal/commit/ec30766a37eecad3678f31815a9f9ef95bbe953e).